doc/source/whatsnew/v1.4.0.rst

```diff
@@ -78,7 +92,45 @@ Optional libraries below the lowest tested version may still work, but are not c
 +-----------------+-----------------+---------+
 | Package         | Minimum Version | Changed |
 +=================+=================+=========+
-|                 |                 |    X    |
+| beautifulsoup4  | 4.8.2           |    X    |
++-----------------+-----------------+---------+
+| fastparquet     | 0.4.0           |         |
++-----------------+-----------------+---------+
+| fsspec          | 0.7.4           |         |
++-----------------+-----------------+---------+
+| gcsfs           | 0.6.0           |         |
++-----------------+-----------------+---------+
+| lxml            | 4.5.0           |    X    |
++-----------------+-----------------+---------+
+| matplotlib      | 3.3.2           |    X    |
++-----------------+-----------------+---------+
+| numba           | 0.50.1          |    X    |
++-----------------+-----------------+---------+
+| openpyxl        | 3.0.2           |         |
++-----------------+-----------------+---------+
+| pyarrow         | 0.17.0          |         |
```

Comment:
if we dont' have an issue to bump this, let's add one.

Reply:
we have #41329 but is closed. will reopen

Reply:
I'll keep 0.17.0 here, bump this later in #41329.